### DISCUSSION TITLE
Who is the audience? (required Bachelor's)

### DISCUSSION THREAD

**Discussion by MakisH** [the-teachingRSE-project/RSE-Masters#36] [General]
(TODO)

**Comment by juliandehne** [comment]
The current approach is to assume that the Curriculum will be adaptable, following this priority:

- MNT BA (MINT without CS)
- SOCIAL SCIENCE BA 
- CS BA
- HUMANITIES BA
- OTHER

The adaptability is achieved my adding the profiles to the meta.yml of the modules which will be processed by the solver.

**Comment by jpthiele** [comment]
@juliandehne Social Science as second part comes from your background and previous contacts, right?
Otherwise, I would put CS as second since a) there are good contacts b) already some interested parties.

**Comment by juliandehne** [comment]
I think Social Science will be very similar to MNT and that's why, but lets switch the order for the reason you mentioned.

**Comment by CaptainSifff** [comment]
Also note, that Julian has a Notion of social sciences that means quantitative social sciences.

**Comment by juliandehne** [comment]
In order to generalize the audience we should use the definition of "Forschungsformen" by the Wissenschaftsrat: https://www.wissenschaftsrat.de/download/archiv/[2359-12.pdf](https://www.wissenschaftsrat.de/download/archiv/2359-12.pdf?__blob=publicationFile&v=0)?__blob=publicationFile&v=0. We should map them to competency cluster such as rse_software engineering or data_science and provide open example list. For example "begrifflich-theoretische" maps to STEM "gestaltende" maps to CS but also to design.
